landing-page/salon-script.js

```diff
@@ -0,0 +1,11 @@
+
+/* Toggle between adding and removing the "responsive" class to 
+topnav when the user clicks on the icon */
+function myFunction() {
```

Comment:
function names should be more descriptive. what's this function doing?
`function toggleNavBar()`